data/lib/src/network/dio_client.dart

```diff
@@ -73,6 +73,23 @@ class DioClient {
           onSendProgress: onSendProgress,
           onReceiveProgress: onReceiveProgress)
         .then((value) => value.data)
-        .catchError((error) => {throw error});
+        .catchError((error) => throw error);
+  }
+
+  Future<dynamic> delete(
+    String path, {
+    data,
+    Map<String, dynamic> queryParameters,
+    Options options,
+    CancelToken cancelToken,
+  }) async {
+    return await _dio.delete(
+          path,
+          data: data,
+          queryParameters: queryParameters,
```

Comment:
we have `parameters` here, why we need to `concat` it to path?

Reply:
I think queryParameters are not what we are looking for we need path parameters